### PR TITLE
add prune branches feature

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds the 'prune branches' feature to the ide git button. currently in alpha for testing and not available yet.

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ The IDE features some delightful tools and layouts to make it easier for you to
...
-     - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which deletes your local branches and refreshes with the current remote branches, effectively removing the deleted ones.
+3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, creating a merge/pull request when appropriate, or pruning branches deleted from the remote repository.
+   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, prune branches, and change branches.
+     - Use the **Prune branches** option to remove local branches that have already been deleted from the remote repository. Selecting this triggers a [pop-up modal](#prune-branches-modal), where you can confirm the deletion of the specific local branches, keeping your branch management tidy.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
<VersionBlock lastVersion="1.5">
         - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which deletes your local branches and refreshes with the current remote branches, effectively removing the deleted ones.
</VersionBlock>

<VersionBlock firstVersion="1.6">
          - Use the **Prune branches** option to remove local branches that have already been deleted from the remote repository. Selecting this triggers a [pop-up modal](#prune-branches-modal), where you can confirm the deletion of the specific local branches, keeping your branch management tidy. Note to use this button,  make sure you're not a delete branch.
</VersionBlock>
```

File: website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ To understand how to navigate the IDE and its user interface elements, refer to
...
+| **IDE version control** | The IDE version control section and git button allow you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project directly into the IDE. <br /><br /> - Create or change branches, execute git commands using the git button.<br /> - Commit or revert individual files by right-clicking the edited file<br /> - [Resolve merge conflicts](/docs/collaborate/git/merge-conflicts)<br /> - Link to the repo directly by clicking the branch name <br /> - Edit, format, or lint files and execute dbt commands in your primary protected branch, and commit to a new branch.<br /> - Use Git diff view to view what has been changed in a file before you make a pull request.<br /> - Use the **Prune branches** [button](/docs/cloud/dbt-cloud-ide/ide-user-interface#prune-branches-modal) to delete local branches that have been deleted from the remote repository, keeping your branch management tidy. |
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md]:
```suggestion
| **IDE version control** | The IDE version control section and git button allow you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project directly into the IDE. <br /><br /> - Create or change branches, execute git commands using the git button.<br /> - Commit or revert individual files by right-clicking the edited file<br /> - [Resolve merge conflicts](/docs/collaborate/git/merge-conflicts)<br /> - Link to the repo directly by clicking the branch name <br /> - Edit, format, or lint files and execute dbt commands in your primary protected branch, and commit to a new branch.<br /> - Use Git diff view to view what has been changed in a file before you make a pull request.<br /> - From dbt version 1.6 and higher, use the **Prune branches** [button](/docs/cloud/dbt-cloud-ide/ide-user-interface#prune-branches-modal) to delete local branches that have been deleted from the remote repository, keeping your branch management tidy. |
```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md [hunk 3]
```diff
@@ -55,9 +55,21 @@ The IDE features some delightful tools and layouts to make it easier for you to
...
+3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, creating a merge/pull request when appropriate, or pruning branches deleted from the remote repository.
+   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, prune branches, and change branches.
+
+<VersionBlock lastVersion="1.5"> 
+    
+   - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which deletes your local branches and refreshes with the current remote branches, effectively removing the deleted ones.
```
Comment (naomiwritescode) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]: Maybe something here that now specifies this to unmerged local branches. Or even this _could_ be removed? I think prune handles most cases where people would need to use this?

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md [hunk 4]
```diff
@@ -55,9 +55,21 @@ The IDE features some delightful tools and layouts to make it easier for you to
...
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+    
+   - Use the **Prune branches** option to remove local branches that have already been deleted from the remote repository. Selecting this triggers a [pop-up modal](#prune-branches-modal), where you can confirm the deletion of the specific local branches, keeping your branch management tidy. Note to use this button,  make sure you're not a delete branch.
+
```
Comment (naomiwritescode) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
   - Use the **Prune branches** option to remove local branches that have already been deleted from the remote repository. Selecting this triggers a [pop-up modal](#prune-branches-modal), where you can confirm the deletion of the specific local branches, keeping your branch management tidy. Note, you won't be able to prune your current branch.

```

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md [hunk 5]
```diff
@@ -146,38 +158,55 @@ You can open the drawer in multiple ways:
...
 
-- **Commit to a new branch** &mdash; Edit directly on your protected primary git branch and commit those changes to a new branch when ready.
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/create-new-branch.png" width="70%" title="Commit changes to a new branch"/>
+<VersionBlock firstVersion="1.6">
 
-- **Commit Changes modal &mdash;** The Commit Changes modal is accessible via the Git Actions button to commit all changes or via the Version Control Options menu to commit individual changes. Once you enter a commit message, you can use the modal to commit and sync the selected changes.
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/commit-changes-modal.png" width="90%" title="The Commit Changes modal is how users commit changes to their branch."/>
+- #### Prune branches modal
+  The Prune branches modal allows users to delete local branches that have been deleted from the remote repository, keeping your branch management tidy. This is accessible through the **Git actions** button under the [**Version control** menu](#editing-features).
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]:
```suggestion
  The Prune branches modal allows users to delete local branches that have been deleted from the remote repository, keeping your branch management tidy. This is accessible through the **Git actions** button under the [**Version control** menu](#editing-features). Note, you won't be able to prune your current branch.
```

**Comment by mirnawong1** [comment]
thanks so much @naomiwritescode ! i've folded those changes in now 🙏

**Review by naomiwritescode — APPROVED** [review]
🚢

**Comment by mirnawong1** [comment]
shipped to multi-tenant and mc - st at the next release. ok'ed by Greg. merging

**Pull request merged**